docs/README.md

```diff
@@ -1,9 +1,9 @@
 Documentation
 =============
 
-The documentation is automatically generated when the main [`README.md`](../README.md) is changed on the `master` branch.
+The documentation is automatically generated when the main [`README.md`](https://github.com/bgruening/docker-galaxy/blob/main/README.md) is changed on the `main` branch.
```

Comment:
```suggestion
The documentation is automatically generated when the main [`README.md`](../README.md) is changed on the `main` branch.
```

Reply:
With this change, the link in `gh-pages` branch to README file is not working anymore as there isn't any README file in that branch.

Reply:
oh, ok, it will not follow relative links, interesting ... thanks